helmclient.go

```diff
@@ -272,8 +272,8 @@ func (c *Client) getReleaseHistory(ctx context.Context, releaseName string) (*Re
 		}
 	}
 
-	if len(resp.Releases) > 1 {
-		return nil, microerror.Maskf(tooManyResultsError, "%d releases found, expected 1", len(resp.Releases))
+	if len(resp.Releases) == 0 {
```

Comment:
👍

Reply:
Too many results cannot happen any more with fake client, it respects max history limit https://github.com/helm/helm/commit/ff3dc9dd33f3a31b6bbc3681d2853c03cd57ebdf#diff-ff36f764520e39db4e97a207158b51b2

I assume it was never possible with real client/tiller, that they respect the limit set in release history request, that we had it just because of bug in FakeClient 2.14 and older.

Reply:
I don't remember the details but that sounds totally reasonable. 👍